src/containers/VisitPage.js

```diff
@@ -62,37 +52,19 @@ class VisitPage extends Component {
       userChanged:           {},
       snippets:              props.snippets,
     };  // end this.state {}
+  };  // End constructor()
 
-    this.steps = [
-      {
-        form:              CurrentBenefitsStep,
-        key:               'currentBenefits',
-        updateClientValue: this.changeCurrent,
-      },
-      {
-        form:              HouseholdStep,
-        key:               'household',
-        updateClientValue: this.changeCurrent,
-      },
-      {
-        form:              CurrentIncomeStep,
-        key:               'currentIncome',
-        updateClientValue: this.changeCurrent,
-      },
-      {
-        form:              CurrentExpensesStep,
-        key:               'currentExpenses',
-        updateClientValue: this.changeCurrent,
-      },
-      {
-        form:              PredictionsStep,
-        key:               'predictions',
-        updateClientValue: this.changeFuture,
-      },//,
-    //  { title: 'Graphs', form: ResultsGraph }
-    ];  // end this.steps {}
+  componentDidMount() {
+    // If the user has refreshed the page while they were on a later step, then we will
+    // go back to that step, but since the client data is currently stored in memory, the
```

Comment:
'isn't stored in memory', I think

Reply:
This also makes it sound like returning to the current step is the currently expected behavior.

Reply:
The intent was to say "the way we currently store client data is in memory, so if you leave the page, it'll be gone". So the intent of this code is to redirect the user to the first step so that they can re-enter the data. I can try to reword it if it's not clear enough.

Reply:
It's not blocking, so we can deal with it another time, but I found it unclear and rewording was what I was trying to describe.